features/gba-dev/uitsluiting-kiesrecht.feature

```diff
@@ -0,0 +1,20 @@
+# language: nl
+
+Functionaliteit: Uitsluiting kiesrecht
+
+
+	@gba
+	Abstract Scenario: Geen aanduiding uitgesloten kiesrecht (38.10) wel een einddatum uitsluiting kiesrecht
+	  Gegeven het systeem heeft een persoon met de volgende gegevens
+		| naam                        | waarde     |
+		| burgerservicenummer         | 999990001  |
+    En de persoon heeft de volgende kiesrecht gegevens
+    | einddatum uitsluiting kiesrecht (38.20) | 20300101 |
```

Comment:
het lijkt mij het handigste om één van de twee stijlen te kiezen die we nu gebruiken:

```
| naam                                    | waarde   |
| einddatum uitsluiting kiesrecht (38.20) | 20300101 |
```
of
```
| einddatum uitsluiting kiesrecht (38.20) |
| 20300101                                |
```

Reply:
was niet bedoeld als nieuwe stijl, gewoon de naam-waarde rij vergeten te kopiëren. Nu toegevoegd.